api/rds.proto

```diff
@@ -48,7 +56,7 @@ message WeightedCluster {
   message ClusterWeight {
     // Name of the upstream cluster. The cluster must exist in the cluster
     // manager configuration.
-    google.protobuf.StringValue name = 1;
+    string name = 1;
```

Comment:
I don't think there are settings in which empty string is a valid option, so I'm not sure if we need to differentiate between unset and empty string values.

Also, see https://github.com/lyft/envoy/issues/773.